osu.Game/Screens/Edit/Compose/Components/BlueprintContainer.cs

```diff
@@ -123,12 +123,20 @@ private void addBlueprintFor(DrawableHitObject hitObject)
             if (blueprint == null)
                 return;
 
-            blueprint.Selected += onBlueprintSelected;
-            blueprint.Deselected += onBlueprintDeselected;
-            blueprint.SelectionRequested += onSelectionRequested;
-            blueprint.DragRequested += onDragRequested;
+            if (hitObject.IsLoaded)
```

Comment:
comment plz

Reply:
Turns out this isn't actually required anymore due to https://github.com/ppy/osu/pull/6527/commits/510ce9345f79b3deee4f6ed1fc72e46ebed0ed0e